projects/ad9739a-fmc-ebz/src/ad9739a_fmc_ebz.c

```diff
@@ -219,8 +221,8 @@ int main(void)
 	iio_axi_dac_get_dev_descriptor(iio_axi_dac_desc, &dev_desc);
 
 	struct iio_data_buffer read_buff = {
-		.buff = (void *)DAC_DDR_BASEADDR,
-		.size = 0xFFFFFFFF,
+		.buff = (void *)dac_buffer,
```

Comment:
Why it has to be used in ad9739a_dmac_init_param also? Maybe some future commit is necessary.

Reply:
I forgot :)